### PR TITLE
ref(face-expressions) refactor face landmarks namings

### PR DESCRIPTION
**Pull request by quitrk**

### CODE REVIEW

File: modules/xmpp/xmpp.js [hunk 1]
```diff
@@ -915,22 +915,22 @@ export default class XMPP extends Listenable {
...
+    sendFaceExpressionEvent(roomJid, payload) {
         // no speaker stats component advertised
         if (!this.speakerStatsComponentAddress || !roomJid) {
             return;
         }
 
         const msg = $msg({ to: this.speakerStatsComponentAddress });
 
-        msg.c('facialExpression', {
+        msg.c('faceExpression', {
```
Comment (gabiborlea) [inline modules/xmpp/xmpp.js:930]: You also have to change this in prosody

Comment (quitrk) [reply modules/xmpp/xmpp.js:930]: that's a different pr

**Review by saghul — DISMISSED** [review]

**Comment by saghul** [comment]
Jenkins please test this please.

**Comment by quitrk** [comment]
Jenkins please test this please.

**Review by saghul — APPROVED** [review]

**Pull request merged**